tests/index.js

```diff
@@ -66,6 +87,15 @@ describe('rule documentation files have the correct content', () => {
         if (expectedNotice === 'deprecated' && documentLines[currentLineNumber + 1] !== MESSAGES[expectedNotice] && documentLines[currentLineNumber + 1].startsWith(MESSAGES[expectedNotice])) {
           // Allow additional rule-specific information at the end of the deprecation notice line.
           assert.ok(true, `includes ${expectedNotice} notice`);
+        } else if (expectedNotice === 'configs') {
+          // Check that the rule specifies its configs.
+          const configsOn = getConfigsForRule(ruleName);
```

Comment:
```suggestion
          const configsOn = getConfigsForRule(ruleName, true);
```
which is better anyways; an optional boolean set to "false", and omitting it, should always give the same result.